dll/cpl/desk/lang/tr-TR.rc

```diff
@@ -4,7 +4,7 @@
  * PURPOSE:     Turkish resource file
  * TRANSLATORS: Copyright 2013-2020 Erdem Ersoy (eersoy93) <erdemersoy@erdemersoy.net>
  *              Copyright 2021-2023 Süleyman Poyraz <zaryob.dev@gmail.com>
- *              Copyright 2024 Ethem Çılgın <ethemrty@gmail.com>
+ *              Copyright 2024 Ethem De Santa <ethemrty@gmail.com>
```

Comment:
`Ethem Çılgın` looks more like a real turkish name, why you reverted that? We only allow real names!

Reply:
> `Ethem Çılgın` looks more like a real turkish name, why you reverted that? We only allow real names!

sorry

Reply:
Thanks